libs/langchain/langchain/vectorstores/redis/filters.py

```diff
@@ -191,21 +203,19 @@ class RedisNum(RedisFilterField):
         RedisFilterOperator.GE: ">=",
     }
     OPERATOR_MAP: Dict[RedisFilterOperator, str] = {
-        RedisFilterOperator.EQ: "@%s:[%f %f]",
-        RedisFilterOperator.NE: "(-@%s:[%f %f])",
-        RedisFilterOperator.GT: "@%s:[(%f +inf]",
-        RedisFilterOperator.LT: "@%s:[-inf (%f]",
-        RedisFilterOperator.GE: "@%s:[%f +inf]",
-        RedisFilterOperator.LE: "@%s:[-inf %f]",
+        RedisFilterOperator.EQ: "@%s:[%s %s]",
+        RedisFilterOperator.NE: "(-@%s:[%s %s])",
+        RedisFilterOperator.GT: "@%s:[(%s +inf]",
+        RedisFilterOperator.LT: "@%s:[-inf (%s]",
+        RedisFilterOperator.GE: "@%s:[%s +inf]",
```

Comment:
Does redis support non-floats here too? Just curious if there were issues leading to this change

Reply:
Yes! This should be fully defined by the user because it could be int or float, and depends on their underlying data or use case. The string formatting should be "dumb" in this sense.